docs/data-model.asciidoc

```diff
@@ -175,6 +175,125 @@ For a full list of tracked metrics, see the relevant agent documentation:
+* `service.runtime.name`: The runtime name of the service that served the transaction, for example `jRuby`
+* `service.runtime.version`: The runtime version that served the transaction
+* `host.hostname`: The hostname of the service that served the transaction
+* `host.os.platform`: The platform name of the service that served the transaction, for example `linux`
+* `container.id`: The container ID of the service that served the transaction
+* `kubernetes.pod.name`: The name of the Kubernetes pod running the service that served the transaction
+* `cloud.provider`: The cloud provider hosting the service instance that served the transaction
+* `cloud.region`: The cloud region hosting the service instance that served the transaction
+* `cloud.availability_zone`: The cloud availability zone hosting the service instance that served the transaction
+* `cloud.account.id`: The cloud account id of the service that served the transaction
+* `cloud.account.name`: The cloud account name of the service that served the transaction
+* `cloud.machine.type`: The cloud machine type or instance type of the service that served the transaction
+* `cloud.project.id`: The cloud project identifier of the service that served the transaction
+* `cloud.project.name`: The cloud project name of the service that served the transaction
+* `cloud.service.name`: The cloud service name of the service that served the transaction
+* `faas.coldstart`: Whether the _serverless_ service that served the transaction had a cold start
+* `faas.trigger.type`: The trigger type that the function / lambda was executed by of the service that served the transaction
+--
+
+The `@timestamp` field of these documents holds the start of the aggregation interval.
+
+[float]
+===== Service-destination metrics
+
+To power {kibana-ref}/xpack-apm.html[APM app] visualizations,
+APM Server aggregates span events into "service destination" metrics.
+
+*`span.destination.service.response_time.count`* and *`span.destination.service.response_time.sum.us`*::
++
+--
+These metrics measure the count and total duration of requests from one service to another service.
+These are used to calculate the throughput and latency of requests to backend services such as databases in
+{kibana-ref}/service-maps.html[Service maps].
+
+These metric documents can be identified by searching for `metricset.name: service_destination`.
+
+You can filter and group by these dimensions:
+
+* `span.destination.service.resource`: The destination service resource, for example `mysql`
+* `event.outcome`: The outcome of the operation, for example `success`
+* `agent.name`: The name of the APM agent that instrumented the operation, for example `java`
+* `service.name`: The name of the service that made the request
+* `service.environment`: The environment of the service that made the request
+--
+
+The `@timestamp` field of these documents holds the start of the aggregation interval.
+
+[float]
+==== Example metric document
```

Comment:
Seeing this, I'm realizing that the other [APM events](https://www.elastic.co/guide/en/apm/guide/current/data-model.html) (transactions, spans, and errors) are missing their [sample documents](https://github.com/elastic/apm-server/tree/main/docs/data/elasticsearch/generated). These used to live [here](https://www.elastic.co/guide/en/apm/guide/7.17/transaction-indices.html).

Would you mind opening a documentation issue in this repo to get them added?